frontend/src/components/RecordsList.tsx

```diff
@@ -217,6 +223,9 @@ export default function RecordsList({
                     refetchWhereVariables={samplesRefetchWhereVariables}
                     setUnsavedChanges={setUnsavedChanges}
                     exportFileName={`${samplesQueryParam}.tsv`}
+                    sampleKeyForUpdate={sampleKeyForUpdate}
+                    userEmail={userEmail}
+                    setUserEmail={setUserEmail}
```

Comment:
Note that these props get passed down from the parent component all the way down to the child `UpdateModal` component, which is the last step in the frontend before changes are submitted to the backend (GraphQL server).